CMakeLists.txt

```diff
@@ -8,12 +8,13 @@ cmake_minimum_required (VERSION 2.8)
 project (edda)
 
 ### Provide options that the user can optionally select.
-option (EDDA_BUILD_MODELER "Build the data modeler programs." OFF)
+option (EDDA_BUILD_MODELING "Build distribution modeling tools." OFF)
```

Comment:
I kept changing these lines back.  Please recover this file back to previous and push again.